src/lua/uri.lua

```diff
@@ -212,10 +212,15 @@ local function params(opts)
     return query_string
 end
 
+local function values(...)
+    return {...}
+end
```

Comment:
The purpose of the function should be clarified, it is not obvious from the surrounding code.